src/sentry/static/sentry/app/components/flowLayout.tsx

```diff
@@ -1,30 +1,29 @@
-import PropTypes from 'prop-types';
 import styled from 'react-emotion';
 
-const FlowLayout = styled('div')`
-  display: flex;
-  flex: 1;
-  align-items: center;
-  flex-direction: ${p => (p.vertical ? 'column' : null)};
-  justify-content: ${p => (p.center ? 'center' : null)};
-  overflow: ${p => (p.truncate ? 'hidden' : null)};
-`;
-
-FlowLayout.propTypes = {
```

Comment:
@matejminar It'll be a good idea to keep `FlowLayout.propTypes` here since other jsx files still consume this component:

```
$ ag flowLayout
docs-ui/components/flowLayout.stories.js
5:import FlowLayout from 'app/components/flowLayout';

src/sentry/static/sentry/app/components/shareIssue.jsx
13:import FlowLayout from 'app/components/flowLayout';

src/sentry/static/sentry/app/views/organizationGroupDetails/groupMerged/mergedItem.jsx
10:import FlowLayout from 'app/components/flowLayout';

src/sentry/static/sentry/app/views/organizationGroupDetails/groupSimilar/similarToolbar.jsx
10:import FlowLayout from 'app/components/flowLayout';

src/sentry/static/sentry/app/views/organizationGroupDetails/groupSimilar/similarItem.jsx
13:import FlowLayout from 'app/components/flowLayout';
```

------

We can safely remove them once this component is exclusively consumed by TypeScript files.

Reply:
ok, thanks, added them back 👍